src/compiler/types.ts

```diff
@@ -1048,10 +1057,16 @@ namespace ts {
         expression: Expression;
     }
 
+    // The reason we create this interface so that JSXAttributes and ObjectLiteralExpression interface can extend out of it.
+    // JSXAttributes differs from normal ObjectLiteralExpression in that JSXAttributes can only take JSXAttribute or JSXSpreadAttribute

```

Comment:
differs -> differ
Add an 's' to the end of `ObjectLiteralExpression`, `JSXAttribute`, `JSXSpreadAttribute`, etc.


What do you mean by "take"?